README.md

```diff
@@ -27,6 +27,9 @@ To learn about limitations and differences between Cloud Storage FUSE and POSIX
 
 For information about pricing for Cloud Storage FUSE, see https://cloud.google.com/storage/docs/gcs-fuse#charges.
 
+## Agreement on Flag Compatibility
+`Experimental` and `hidden` flags are subject to changed/removed even across minor releases and have no SLA.
```

Comment:
Also, users dont even know about hidden flags. It seems unnecessary to mention here.
Experimental seems like a note we could add in the flags page or in the dev guide.